app/services/sso/jwt.rb

```diff
@@ -42,5 +48,19 @@ def bearer_token_set?
     def no_issuer?
       jwt_token.decoded_payload.present? && !jwt_token.decoded_payload.key?('iss')
     end
+
+    def controller_name
+      controller.controller_permission
+    end
+
+    def valid_scope?(scope)
+      return true if scope.empty?
+
+      if scope.is_a? Array
+        scope.include?(controller_name)
+      else
+        scope == controller_name
```

Comment:
Should the scope be a whole controller? or perhaps the scope should be a specific controller action?

Reply:
Well, we've never talked about the option with restricting scope to actions too, and I don't have any strong feelings about it, so maybe it's up to community what they think.

We still check user's privileges though, for action and for controller, this haven't been changed by this PR,
so I'm not sure if there is any additional value to have it in token too, maybe the restriction by controller is enough.

Plus I would say that nobody (please correct me if I'm wrong here) never requested restricting JWT scope until Global Registration feature came out, so giving it time and see how users will use it and they will think, may won't be a bad idea.